docs/users.rst

```diff
@@ -17,11 +17,68 @@ System Requirements
 * To verify your JRE: https://www.java.com/en/download/help/version_manual.xml
 * To download JRE 1.8 (AKA JRE 8): http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html
 * We also recommend downloading the VIA vizualization app and obtaining a free or paid license: https://simunto.com/via/
-* Git installed
+* Git and Git-LFS
 
+Prerequisites :
+^^^^^^^^^^^^^^^
 
-Installing
-^^^^^^^^^^
+**Install Java**
+
+BEAM requires Java 1.8 JDK / JRE to be installed. If a different version of java is already installed on your system, please upgrade the version to 1.8.
+See this `link <https://www.java.com/en/download/help/version_manual.xml>`_ for steps to check the current version of your JRE.
+
+If java is not already installed on your system , please follow this `download manual <https://www.java.com/en/download/manual.jsp>`_ to install java on your system.
+
+Please note that BEAM is currently compatible only with Java 1.8 and is not compatible with any of the older or recent versions.
+
+**Install Gradle**
+
+BEAM uses `gradle <https://gradle.org>`_ as its build tool. If gradle is not already installed, check this `gradle installation guide <https://gradle.org/install>`_ for steps on how to download and install gradle.
+Once gradle is successfully installed , verify the installation by running the command
+
+.. code-block:: bash
+
+    gradle
+
+
+GIT-LFS Configuration
+^^^^^^^^^^^^^^^^^^^^^
+
+The installation process for git-lfs client(`v2.3.4`_, latest installer has some issue with node-git-lfs) is very simple. For detailed documentation please consult `github guide`_ for Mac, windows and Linux.
+
+.. _v2.3.4: https://github.com/git-lfs/git-lfs/releases/tag/v2.3.4
+.. _github guide: https://help.github.com/articles/installing-git-large-file-storage/
+
+To verify successful installation, run following command::
+
+    $ git lfs install
+    Git LFS initialized.
+
+To confirm that you have installed the correct version of client run the following command::
+
+   $ git lfs env
+
+It will print out the installed version, and please make sure it is `git-lfs/2.3.4`.
+
+To update the text pointers with the actual contents of files, run the following command (if it requests credentials, use any username and leave the password empty)::
+
+   $ git lfs pull
+   Git LFS: (98 of 123 files) 343.22 MB / 542.18 MB
+
+
+**Installing git lfs on windows :**
```

Comment:
I'm not sure it's worth mentioning, but I was having problems with this and finally traced my issue down to that I needed to replace the git-lfs.exe in the C:\Program Files\Git\mingw64\bin. Just replacing it in C:\Program Files\Git\cmd wasn't enough if you use the git bash

Reply:
@colinsheppard wanted to mention this specifically for windows users.

Reply:
@JustinPihony can you please mention the steps we need to correct or add wrt to the windows installation here . I do not use windows so I am not sure where and what exactly should I change the guide

Reply:
Just a footnote after the steps. Something like:

* Another alternative to above is to get the old git-lfs.exe from the release archives and replace the executable found in `[INSTALL PATH]\mingw6\bin` and `[INSTALL PATH]\cmd`, where the default installation path is `C:\Program Files\Git`

Reply:
done